packages/slate-react/test/index.js

```diff
@@ -9,7 +9,7 @@ describe('slate-react', () => {
   describe('Editable', () => {
     describe('decorate', () => {
       // stub out some DOM stuff to avoid crashes
-      before(() => {
+      beforeEach(() => {
```

Comment:
Please can you clarify this change? 

Also @timbuckley mocha config still runs this test-case, do we need to make changes to `test:mocha` command?

Reply:
`before` is not a recognized by jest, where `beforeEach` is https://jestjs.io/docs/setup-teardown

For mocha: I have changed `./packages/*/test/**/*.{js,ts}`to `./packages/{slate,slate-history,slate-hyperscript}/test/**/*.{js,ts}`